target/linux/octeon/base-files/lib/preinit/03_rename_interfaces

```diff
@@ -0,0 +1,26 @@
+. /lib/functions.sh
+
+rename_interface_by_mac() {
+	local interface_name="$1"
+	local interface_mac="$2"
+	for device in $(ls -1 /sys/class/net) ; do
+		[ -d "/sys/class/net/$device/device" ] && \
+		[ -f "/sys/class/net/$device/address" ] && \
+		[ "$(cat /sys/class/net/$device/address)" = "$interface_mac" ] && {
+			ip link set "$device" name "$interface_name"
+		}
+	done
+}
+
+rename_interfaces() {
```

Comment:
Consider changing to "octeon_rename_interfaces" ...

Reply:
renamed to `octeon_rename_interfaces`